charts/bitbucket-runner-autoscaler/templates/cleaner.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bitbucket-runner-controller-cleaner
```

Comment:
I want to use a common Helm naming pattern here, using the `_helper` file to compose the name using the Helm release name. Otherwise, the chart can be installed only once, which might not be desired.